pom.xml

```
@@ -90,6 +90,10 @@
     <node.version>you-must-override-the-node.version-property</node.version>
     <npm.version>you-must-override-the-npm.version-property</npm.version>
     <yarn.version>you-must-override-the-yarn.version-property</yarn.version>
+    <!-- can be set to the empty string or install for yarn2 -->
+    <!-- ensure only one concurrent 'yarn install' -->
+    <!-- when yarn cache is empty, multiple yarns performing network fetches frequently results in opaque errors -->
+    <yarn.args>---mutex network</yarn.args>
```

```suggestion
    <yarn.args>--mutex network</yarn.args>
```

Why are there three dashes in the new version of this property but only two dashes in the old version? Please provide a *Testing done* section explaining how you have tested this change, including proof that a computer has executed the changed lines.